pkg/services/accesscontrol/models.go

```diff
@@ -76,11 +76,10 @@ const (
 	ActionLDAPStatusRead = "ldap.status:read"
 
 	// Global Scopes
-	ScopeUsersAll  = "users:*"
-	ScopeUsersSelf = "users:self"
+	ScopeGlobalUsersAll = "global:users:*"
 
-	ScopeOrgAllUsersAll     = "org:*/users:*"
-	ScopeOrgCurrentUsersAll = "org:current/users:*"
+	ScopeUsersSelf = "users:self"
```

Comment:
This one does not seem to be used?

Reply:
not yet, we could remove that, but I'll do that a bit later 🙂